test/integration/attributes-work/default.rb

```diff
@@ -0,0 +1 @@
+# will put a test that makes sure a file is there
```

Comment:
Because the profile that actually consumes and uses the attribute is elsewhere (and run by the audit cookbook, not TK), and this is empty, TK won't actually be doing any real testing of this new test suite. To make matters worse, Chef will still exit with a zero exit code even if the audit cookbook failed to scan properly or if there are compliance failures. 🙂 

Can I recommend we add a test here that reads in the JSON outputted by the audit cookbook, and ensures that our test succeeds?